ops/pebble.py

```diff
@@ -2032,6 +2032,9 @@ def _wait_change_using_wait(self, change_id: ChangeID, timeout: Optional[float])
             except TimeoutError:
                 # Catch timeout from wait endpoint and loop to check deadline
                 pass
+            except socket.timeout:
```

Comment:
Note that you can use the following syntax to catch multiple exception types:

```suggestion
            except (TimeoutError, socket.timeout):
```

However, that makes me think: `TimeoutError` (which is `pebble.TimeoutError` here) inherits from the builtin `TimeoutError`, and [`socket.timeout`](https://docs.python.org/3/library/socket.html#socket.timeout) is a "deprecated alias of [the builtin] TimeoutError". So I think we can `import builtins` and just catch a single type here:

```
except builtins.TimeoutError:
```

Separately, I regret naming the pebble error `TimeoutError` in pebble.py, instead of just using the builtin exception type, as it's too confusing ... but it's too late to change that. :-)

Reply:
I think `except builtins.TimeoutError` alone won't cache it if a `socket.timeout` exception is raised for Python <3.10 because only since 3.10 it becomes an alias, right? 

In order to make it work for all versions of Python work, I will explicitly put `except (builtins.TimeoutError, socket.timeout)` here.